crates/router/src/core/payments/operations/payment_confirm.rs

```diff
@@ -1433,6 +1441,28 @@ impl<F: Clone + Sync> UpdateTracker<F, PaymentData<F>, api::PaymentsRequest> for
             ),
             None => (None, None, None),
         };
+        let order_tax_amount =
+            payment_data
+                .payment_intent
```

Comment:
why are we fetching this again? we have already set this in payment_attempt in get trackers right?